.bumpversion.cfg

```diff
@@ -0,0 +1,30 @@
+[bumpversion]
+current_version = 1.1.0-internal-6
```

Comment:
Is the plan to get rid of this?

Reply:
This is where the source of truth for the package versions is now specified. Were you referring to the old settings.cfg in `tools` where we used to specify the package version?

Reply:
Sorry. My comment was very vague. I was talking about the "internal-6" piece. What we've been using in lieu of a build number.

Reply:
I haven't made any changes to the versioning scheme. As we talked offline, we can change the versioning process so that every single commit does not have to bump the build version. Instead, we could automatically mark pre-release versions with a commit hash. I wasn't planning on implementing this process here.

Reply:
Sounds good. Mind filing an issue to track that so we don't forget?

Reply:
Definitely. I filed `PYT-1053: Simplify Virtualization SDK pre-release version format`.